src/test/java/seedu/plannermd/testutil/PlannerMdBuilder.java

```diff
@@ -23,8 +23,8 @@ public PlannerMdBuilder(PlannerMd plannerMd) {
     /**
      * Adds a new {@code Person} to the {@code PlannerMd} that we are building.
```

Comment:
Perhaps this should be Patient?
```suggestion
     * Adds a new {@code Patient} to the {@code PlannerMd} that we are building.
```